src/components/panel/panel.js

```diff
@@ -129,7 +129,7 @@ angular
  *     Only disable if focusing some other way, as focus management is required for
  *     panels to be accessible. Defaults to true.
  *
- * @returns {$MdPanelReference} panelRef
+ * @returns {$mdPanelReference} panelRef
```

Comment:
Why change this from PascalCase? I think the ideal is just `MdPanelRef` (without the `$`)